src/tests/stdlib.rs

```diff
@@ -18,11 +18,32 @@ fn stdlib_test(input: &str) {
     let code =
         fs::read_to_string(input).unwrap_or_else(|_| panic!("Failed to open {input} test file"));
 
-    let output = match Path::new(&input.replace(".ab", ".output.txt")).exists() {
-        true => fs::read_to_string(input.replace(".ab", ".output.txt"))
-            .unwrap_or_else(|_| panic!("Failed to open {input}.output.txt file")),
-        _ => "Succeded".to_string(),
-    };
+    let mut is_output = false;
+    let mut output = "".to_owned();
+    for line in code.lines() {
+        if line.starts_with("// Output") {
+            is_output = true;
+            continue;
+        } else if line.is_empty() && is_output {
+            is_output = false;
+            break;
+        }
+
+        if is_output {
+            if ! output.is_empty() {
+                output.push_str("\n");
+            }
+            output.push_str(&line.replace("//", "").trim());
+        }
+    }
+
+    if output.is_empty() {
+        output = match Path::new(&input.replace(".ab", ".output.txt")).exists() {
+            true => fs::read_to_string(input.replace(".ab", ".output.txt"))
+            .expect(&format!("Failed to open {input}.output.txt file")),
+            _ => "Succeded".to_string()
```

Comment:
This will help to see that this line is not part of the pattern matching:

```suggestion
            true => fs::read_to_string(input.replace(".ab", ".output.txt"))
                .expect(&format!("Failed to open {input}.output.txt file")),
            _ => "Succeded".to_string()
```